test/analytics/plugins/analytics_ftr_helpers/package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "analytics_ftr_helpers",
+  "name": "@kbn/analytics-ftr-helpers-plugin",
```

Comment:
@afharo could you please take a look at these changes? They look fine to me but there may be consequences that aren't obvious to me w.r.t EBT

Reply:
yeah! all good here! 👍